.travis.yml

```diff
@@ -10,10 +10,14 @@ before_install:
   - export PATH=$PATH:$PWD/linux-amd64/
   - glide install
 install:
-  - go install -v $(glide novendor)
-  - go get -v golang.org/x/tools/cmd/cover
-  - go get -v github.com/golang/lint/golint
-  - go get -v github.com/davecgh/go-spew/spew
+  - go get -v github.com/alecthomas/gometalinter
+  - gometalinter --install
```

Comment:
uh, you can't remove the glide install :)

I'd also move all of these gometalinter installing stuff into before_install because it's required test tooling, not building/installing dcrwallet itself.